docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+For example, if you wanted to add a page under `Getting Started` and the file is called `example-tutorial.md`, you would save the file in the folder called `/getting-started/`. Next, you would modify the config file like the example below:
+
+`Path: ./strapi/docs/.versions/3.0.0-beta.x.js/`
+
+```js
+module.exports = [
+  {
+    collapsable: false,
+    title: '🚀 Getting started',
+    children: [
+      '/3.0.0-beta.x/getting-started/introduction',
+      '/3.0.0-beta.x/getting-started/install-requirements',
+      '/3.0.0-beta.x/getting-started/quick-start',
+      '/3.0.0-beta.x/getting-started/quick-start-tutorial',
+      '/3.0.0-beta.x/getting-started/example-tutorial',  // <- Added here
+    ],
+  },
+
+```
+
+For example, if you now wanted the **left-hand** menu to have a link to **Example Tutorial**, you would need to add an `h1` tag with `Example Tutorial`, like the below example:
+
+```md
+# Example Tutorial
+
+This is an Example Tutorial....
+```
+
+You should understand that [VuePress]() automatically turns `h2` and `h3` tags into correctly formed **submenu items** and **nested submenu items**. These display in the left-hand navigation underneath the correct article heading as determined by the `h1` tag on the page.
+
+Now, if you need to add images to the file, there is a folder called `assets`, the `Path: ./3.0.0-beta.x/assets`. When you add images, these images need are added to a new folder in the `assets` folder. The name of the folder, should match the name of the file and be placed in a matching directory to the directory of the file.
+
+For example, the `example-tutorial.md` file is located in the `/getting-started/` directory.
+
+```js
+module.exports = [
+  {
+    collapsable: false,
+    title: '🚀 Getting started',
+    children: [
+      '/3.0.0-beta.x/getting-started/example-tutorial',  // <- Added here
+    ],
+  },
+
+```
+
+Therefore, images are placed in `./assets/getting-started/example-tuturial/image.png`, like this:
+
+```
+DOCS
+└───.vuepress
+└───.3.0.0-beta.x
+│   └───getting-started
+│   │   └───example-tutorial.md         // Markdown File
+│   │
+│   └───assets                          // Image Assets
+│   │   └───getting-started             // Directory of the Markdown file
+│   │       └───example-tutorial        // Name of the Markdown File
+│   │           image1.jpg
+│   │           image2.jpg
+│   │           image2.jpg
+│   │
+│   └───etc
+```
+
+**Note:** Images are not linked outside the documentation, but **can** link to an image saved in a different directory. Mainly if the image already exists.
+
+These above guidelines illustrate how to add pages, images, and the structure of the documentation. The documentation is in a language called [Markdown](https://guides.github.com/features/mastering-markdown/), you can do this [Markdown Tutorial](https://www.markdowntutorial.com) to learn how to use Markdown.
```

Comment:
The documentation is written in [Mardown](link), you can follow this [Md tuto](link)

Reply:
I clarified this sentence.